src/Docfx.Build.Common/ModelAttributeHandlers/Handlers/MarkdownContentHandler.cs

```diff
@@ -21,15 +21,8 @@ public object Handle(object obj, HandleModelAttributesContext context)
             return null;
         }
 
-        if (context == null)
-        {
-            throw new ArgumentNullException(nameof(context));
-        }
-
-        if (context.Host == null)
-        {
-            throw new ArgumentNullException(nameof(context.Host));
-        }
+        ArgumentNullException.ThrowIfNull(context);
+        ArgumentNullException.ThrowIfNull(context.Host);
```

Comment:
Sorry for confusing by following statements.
> These change will not affects application behaviors.

I've slightly modified some behaviors. that don't affect application behaviors.
It might be better to indicate  what I've changed manually with following steps.

1. Apply `CA1510`  code fixture by `dotnet format`. and commit changes.
2. Edit code that require manual modification.